src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ReceivedMessagesHealthController.java

```diff
@@ -114,6 +120,12 @@ private boolean isDateWithinWorkingHours(LocalDateTime localDateTime) {
     }
 
     private boolean isNotEnabledForEnvironment(String env) {
+        if (ENV_AAT.equals(env)) {
+            URI currentUri = ServletUriComponentsBuilder.fromCurrentRequestUri().build().toUri();
```

Comment:
could this ever be `null` or empty?
If it can then would it throw an exception?

Reply:
Also should put some logging so we can see it is getting the correct url and excluding staging correctly when running.